packages/mock-account-provider/app/routes/accounts.$accountId.tsx

```diff
@@ -0,0 +1,48 @@
+import { json } from '@remix-run/node'
+import { useLoaderData } from '@remix-run/react'
+import { getAccountTransactions } from '../lib/transactions.server'
+import tableStyle from '../styles/table.css'
+
+type LoaderData = {
+  transactions: Awaited<ReturnType<typeof getAccountTransactions>>
+}
+
+export const loader = async ({ params }) => {
+  return json<LoaderData>({
+    transactions: await getAccountTransactions(params.accountId)
+  })
+}
+
+export default function Transactions() {
+  const { transactions } = useLoaderData() as LoaderData
+  return (
+    <main>
+      <h1>Transactions</h1>
+      <table>
+        <tr>
+          <th>Date</th>
+          <th>Type</th>
+          <th>Amount</th>
+          <th>Description</th>
+        </tr>
+        {transactions.map((trx) => (
+          <tr key={trx.id}>
+            <td>{trx.createdAt}</td>
+            <td>{trx.sendAmountValue ? 'Outgoing' : 'Incoming'}</td>
```

Comment:
this is a bit implicit determination, IMO would be better to programatically set a `type` on each payment or something similar